csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -672,7 +672,7 @@ The Common Platform Enumeration (CPE) attribute refers to a method for naming pl
 
 ##### 3.1.3.3.2 Full Product Name Type - Product Identification Helper - Hashes
 
-List of `hashes` of value type `array` holding at least one item contains a list of cryptographic hashes usable to identify files.
+List of hashes (`hashes`) of value type `array` holding at least one item contains a list of cryptographic hashes usable to identify files.
```

Comment:
I will merge but later we might consider not doubling identical terms (when the English language term is the key, one instance could be left out to ease the reader - either the English or the "synthetic" term - I would maybe keep the English term)